kubeflow/trainer/rhai/utils.py

```
@@ -0,0 +1,37 @@
+        return transformers.get_trainer_crd_from_transformers_trainer(
+            runtime,
+            trainer,
+            initializer,
+        )
+
+    else:
+        _raise_unknown_rhai_trainer(trainer)
+
+
+def _raise_unknown_rhai_trainer(trainer: NoReturn) -> NoReturn:
+    raise ValueError(f"Unknown trainer {trainer}.")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)